src/Misc/layoutbin/darwin.svc.sh.template

```diff
@@ -40,6 +42,14 @@ function install()
     cp "${TEMP_PATH}" "${PLIST_PATH}" || "failed to copy plist"
     cp ./bin/runsvc.sh ./runsvc.sh
     chmod 755 ./runsvc.sh
+
+    echo '
+    {
+        "RunAsService": true,
```

Comment:
I moved the properties from .Agent to .Service, but indeed the only important thing is if the file exists. I agree with your comments below.